ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/config/WorkflowConfigParserSpec.scala

```diff
@@ -125,6 +129,7 @@ class WorkflowConfigParserSpec extends AnyWordSpec with Matchers {
               ),
               maxItemCount = Some(700),
               timeoutInSecondsO = None,
+              subscriptionDelay = Some(Duration(7, TimeUnit.MINUTES)),
```

Comment:
Is there a test to verify this configuration key is optional? `subscriptionDelay = None`

Reply:
added 👍